R2API/ItemAPI.cs

```diff
@@ -329,7 +330,8 @@ public class CustomItem {
             bool hidden,
             bool canRemove,
             UnlockableDef unlockableDef,
-            ItemDisplayRule[]? itemDisplayRules) {
+            ItemDisplayRule[]? itemDisplayRules,
+            ItemTierDef itemTierDef) {
```

Comment:
You were asking if this was a breaking change:
This is a breaking change, the method signature has changed.

Reply:
Truthfully, i think i forgot to add "= null" to the itemTierDef (AFAIK that wont break mods, we had a similar talk before on #324

Reply:
That would still change the method signature, and they would need to be recompiled against the new binary

Reply:
Agreed, i thought this would've worked fine as the example o added above regarding dotAPI was just making the associated buff def field default to null, afaik no mods that used dotApi broke from that change, albeit that might be because here I'm adding a new argument instead of defaulting an argument to null

Reply:
Yes, that is why